### ISSUE TITLE
Show alert controller without title and message if its nil

### ISSUE THREAD

**Issue by disquad** [sberrevoets/SDCAlertView#283]
Hello, I would like to make fully custom contentView inside of alert controller without title and message, but if set title and message to nil its look like

![2018-10-08 5 29 04 pm](https://user-images.githubusercontent.com/6170103/46614918-ad7ee780-cb1f-11e8-9bba-db254bb59078.png)

**Comment by sberrevoets** [comment]
Are you talking about the space at the top?

**Comment by disquad** [comment]
@sberrevoets Hello, thank you for attention. Yeh, I am talking about space at the top

**Comment by sberrevoets** [comment]
Yeah that's a problem, though I can't guarantee when I can have a fix ready for this. If you can somehow use the built-in labels that's probably your best bet.

**Comment by disquad** [comment]
@sberrevoets ah, hope, in the nearest future I will can use it instead mdc modals :)
This library is so nice
Thank you for your work 👍
